src/pal/src/file/path.cpp

```diff
@@ -1425,8 +1500,12 @@ SearchPathW(
             }
     
             /* see if the file exists */
-            WideCharToMultiByte(CP_ACP, 0, CanonicalPath, -1,
-                                AnsiPath, MAX_LONGPATH, NULL, NULL);
+            CanonicalPathLength = (PAL_wcslen(CanonicalPath)+1) * 3;
```

Comment:
Could you please use the constant MaxWCharToAcpLengthRatio instead of just the number 3 here (and at 4 other places)?